docs/infrastructure_and_maintenance/security/security_checklist.md

```diff
@@ -140,6 +140,32 @@ The older UTF8 can lead to truncation with 4-byte characters, like some emoji, w
 
 See [Change from UTF8 to UTF8MB4](update_db_to_2.5.md#change-from-utf8-to-utf8mb4).
 
+### Use restricted DB user
+
+The Data Definition Language (DDL) commands (create, alter, drop, truncate, comment) are only needed for installing and upgrading [[= product_name =]], and not for running it.
+Not granting these rights to web app users reduces the damage that can result from a security breach.
+
+Create a user and grant minimal rights:
+
+=== "MySQL"
+
+    ```sql
+    CREATE USER 'user'@'host' IDENTIFIED BY 'password';
+    GRANT SELECT, INSERT, UPDATE, DELETE ON database_name.* TO 'user'@'host';
```

Comment:
Won't we need TCL commands here, too? COMMIT, ROLLBACK, SET at least?

Reply:
I may have missinterpreted your remark at first. I thought it was about wrapping the commands in a transaction batch.

`SET` could be added in PostgreSQL: https://www.postgresql.org/docs/current/sql-grant.html#SQL-GRANT-DESCRIPTION-OBJECTS

I see no equivalent on MySQL: https://dev.mysql.com/doc/refman/8.4/en/privileges-provided.html